docs/content/Getting-Started/quickstart.md

```diff
@@ -294,7 +294,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8081
+          hostPort: 8094
```

Comment:
This again is the kube `hostPort` that is staying 8081.

Reply:
gotcha - will update - thank you